nrf_security/doc/backend_config.rst

```diff
@@ -1,73 +1,43 @@
-.. _nrf_security_backend_config:
+.. _nrf_security_legacy_config:
 
-Backend configurations and supported features
-#############################################
+Legacy configurations and supported features
+############################################
 
 .. contents::
    :local:
    :depth: 2
 
-This section covers the configurations available when either linking directly to a backend library or when using the Mbed TLS glue layer.
+This section covers the configurations available when using legacy crypto API support.
```

Comment:
legacy needs to be defined.

Reply:
If we have defined it elsewhere we could add a link to what it is from here.

Reply:
Can make this into a link, yes please see the other comment I tagged you with to see if you would like more details on legacy crypto support

Reply:
Fixed with link